anoncreds/protocol/types.py

```diff
@@ -266,6 +295,19 @@ class ClaimRequest(namedtuple('ClaimRequest', 'userId, U, Ur'),
     def __new__(cls, userId, U, Ur=None):
         return super(ClaimRequest, cls).__new__(cls, userId, U, Ur)
 
+    def to_str_dict(self):
```

Comment:
I still have some concerns about backward-compatibility of the changes...
I know it's ugly, but if we add the following check to `crypto_int_to_str`: if it contains 'mod', then let's de-serialize it in the 'old' way. If not - let's use the new way.